### PR TITLE
Add  _.toPairs / _.entires / _.pairs

### PR DESCRIPTION
**Pull request by inuscript**
Append altanative `toPairs` to `Object.entries`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -905,7 +906,7 @@ Retrieves all the names of the object's own enumerable properties.
 
 ### _.values
 
-Retrieve all the names of the object's own enumerable properties.
+Retrieve all the given object's own enumerable property values.
```
Comment (inuscript) [inline README.md]: And sorry, I had mistake this line in #80

File: README.md [hunk 2]
```diff
@@ -927,6 +928,31 @@ Retrieve all the names of the object's own enumerable properties.
 
 **[⬆ back to top](#quick-links)**
 
+### _.toPairs
+
+Retrieve all the given object's own enumerable property `[ key, value ]` pairs.
```
Comment (stevemao) [inline README.md]: Retrieves

File: README.md (hunk 1 above)
Comment (stevemao) [inline README.md]: we recently changed all verbs to third-person singular. So this should start with "Retrieve"

**Comment by stevemao** [comment]
Thanks @inuscript so much :)

File: README.md (hunk 2 above)
Comment (inuscript) [inline README.md]: Oops, I fix it.

**Pull request merged**